src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,352 @@
+  data: Array<DappV1 | DappV2>
+  category: CategoryWithDapps
+}
+
+interface DappFilterChip {
+  chipFilter: DappFilter
+  selectedFilter: DappFilter
+  setFilter: (filter: DappFilter) => void
+  lastChip: boolean
+}
+
+export function DAppsExplorerScreen() {
+  const { t } = useTranslation()
+  const insets = useSafeAreaInsets()
+
+  const sectionListRef = useRef<SectionList>(null)
+  const scrollPosition = useRef(new Animated.Value(0)).current
+  const horizontalScrollView = useRef<ScrollView>(null)
+
+  const onScroll = Animated.event([{ nativeEvent: { contentOffset: { y: scrollPosition } } }])
+  const dispatch = useDispatch()
+  const loading = useSelector(dappsListLoadingSelector)
+  const error = useSelector(dappsListErrorSelector)
+  const categories = useSelector(dappsCategoriesAlphabeticalSelector)
+  const dappFavoritesEnabled = useSelector(dappFavoritesEnabledSelector)
+  const dappsMinimalDisclaimerEnabled = useSelector(dappsMinimalDisclaimerEnabledSelector)
+  const dappList = useSelector(dappsListSelector)
+  const favoriteDappsById = useSelector(favoriteDappIdsSelector)
+  const [selectedFilter, setSelectedFilter] = useState({
+    id: 'all',
+    name: t('dappsScreen.allDapps'),
+  })
+
+  const { onSelectDapp, ConfirmOpenDappBottomSheet } = useOpenDapp()
+  const { onFavoriteDapp, DappFavoritedToast } = useDappFavoritedToast(sectionListRef)
+  const { openSheet, DappInfoBottomSheet } = useDappInfoBottomSheet()
+
+  useEffect(() => {
+    dispatch(fetchDappsList())
+    ValoraAnalytics.track(DappExplorerEvents.dapp_screen_open)
+  }, [])
+
+  return (
+    <SafeAreaView style={styles.safeAreaContainer} edges={['top']}>
+      <DrawerTopBar
+        rightElement={
+          <TopBarIconButton
+            testID="DAppsExplorerScreen/HelpIcon"
+            icon={<Help color={colors.onboardingGreen} />}
+            onPress={openSheet}
+          />
+        }
+        scrollPosition={scrollPosition}
+      />
+      {ConfirmOpenDappBottomSheet}
+      <>
+        {!loading && error && (
+          <View style={styles.centerContainer}>
+            <Text style={fontStyles.regular}>{t('dappsScreen.errorMessage')}</Text>
+          </View>
+        )}
+        {categories.length && (
+          <AnimatedSectionList
+            refreshControl={
+              <RefreshControl
+                tintColor={colors.greenBrand}
+                colors={[colors.greenBrand]}
+                style={{ backgroundColor: colors.light }}
+                refreshing={loading}
+                onRefresh={() => dispatch(fetchDappsList())}
+              />
+            }
+            // @ts-ignore TODO: resolve type error
+            ref={sectionListRef}
+            ListFooterComponent={
+              <>
+                {dappsMinimalDisclaimerEnabled ? (
+                  <Text style={styles.disclaimer}>{t('dappsDisclaimerAllDapps')}</Text>
+                ) : null}
+              </>
+            }
+            ListHeaderComponent={
```

Comment:
this SectionList is a little hard to read when the props are 53 lines long. what do you think about taking the whole horizontally scrolling carousel and turning it into a component / new file?